src/Nancy/project.json

```diff
@@ -7,7 +7,7 @@
   "licenseUrl": "",
 
   "dependencies": {
-    "Microsoft.Framework.Runtime.Interfaces": "1.0.0-beta4"
+    "Microsoft.Framework.Runtime.Interfaces": "1.0.0-beta5"

```

Comment:
It wont compile at all on dnxcore50 at the moment cause of changes and things missing that we have to work through